presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -200,6 +200,11 @@ Specifier Description
 
 .. warning:: The following specifiers are not currently supported: ``%D %U %u %V %X``
 
+.. warning:: Timestamp is truncated to milliseconds
+
+.. note:: ``Fraction of second`` behaves the same way as ``Microseconds`` from the MySQL reference manual (for the ``date_format`` function)
+    Similarly to MySQL, it accepts between 6 and 9 digits of input.
```

Comment:
Actually we're not doing exactly the same thing. We do the same modulo we truncate on 3 digits instead of 6, as MySQL does.
I'll modify docs to reflect this more accurately.